src/main/ApplicationImpl.cpp

```diff
@@ -681,14 +681,7 @@ ApplicationImpl::start()
     }
 
     bool done = false;
-    mLedgerManager->loadLastKnownLedger([this,
-                                         &done](asio::error_code const& ec) {
-        if (ec)
-        {
-            throw std::runtime_error(
-                "Unable to restore last-known ledger state");
-        }
-
+    mLedgerManager->loadLastKnownLedger([this, &done]() {
```

Comment:
this can't fail anymore?

Reply:
yeah, this is a minor cleanup (needed regardless of this change). The only place we had an actual error code was in "repair buckets" work, but that was removed a long time ago in https://github.com/stellar/stellar-core/pull/2232/files#diff-ce04c2790c0d0c23aa7da5c9e2053e7a60f2e286fccb33a1b98dc704340c3f87L325 So we don't need `ec` and the lambda